vars/main.yml

```diff
@@ -6,9 +6,9 @@ core:
 
 ueransim:
   servers:
-    0:
-      - "config/custom-gnb.yaml"
-      - "config/custom-ue.yaml"
+    "0":
```

Comment:
Why does this have to be a string?

Reply:
It makes it easier to loop through and find the corresponding server.

Reply:
Then, why do you use "numbers" in the `gnbsim` repo (https://github.com/gab-arrobo/aether-gnbsim/blob/master/vars/main.yml#L18)? I was thinking that these two (gnbsim, and ueransim) should be similar (preferable, numbers)